packages/types/src/gateway-schema.json

```diff
@@ -149,6 +149,23 @@
         }
       ],
       "description": "This value specifies an HTTP Request size limit for a particular API Definition. Accepts numeric (in bytes) or string values, i.e. 'b' for bytes, 'kb' for kilobytes and 'mb' for megabytes (f.e., '10kb' or '10mb')."
+    },
+    "tracing": {
+      "type": "object",
+      "description": "Configure Jaeger agent endpoint",
+      "additionalProperties": false,
+      "properties": {
```

Comment:
Today we support only Jaeger but in future, we might also support Zipkin. Let's add a "tracer name" property or smth like that.